src/org/dux/cli/DuxBuildTracer.java

```diff
@@ -25,7 +25,7 @@
  * Currently depends on having strace available
  */
 public class DuxBuildTracer {
-    private static final String TMP_FILE = ".dux_out";
+    private static final String TMP_FILE = "trace.out";
```

Comment:
Should probably name the temporary file so that it is hidden on Linux - i.e. so that it starts with a "."